content/graphing/miscellaneous/functions.md

```diff
@@ -224,6 +224,8 @@ The method can be sum/min/max/count/avg and time is in seconds. You can use eith
 
 These functions are only intended for metrics submitted as rates or counters via statsd. These functions will have no effect for other metric types. For more on details about how to use `.as_count()` and `.as_rate()` please see [our blog post][1].
 
+Note: The only available query with `as_count()` is `sum()` (unless using a rollup summary), which is the only mathematical accurate function with such behavior.
```

Comment:
so if you use rollup summary, what other query is available? and what does 'which' refer to? re: the only math accurate function. And what does 'with such behavior' refer to?